### PR TITLE
NOISSUE - Added normalizer service to run script

### PR DESCRIPTION
**Pull request by mteodor**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -47,11 +47,20 @@ MF_HTTP_ADAPTER_LOG_LEVEL=info MF_HTTP_ADAPTER_PORT=8185 MF_THINGS_URL=localhost
 ###
 MF_WS_ADAPTER_LOG_LEVEL=info MF_WS_ADAPTER_PORT=8186 MF_THINGS_URL=localhost:8183 $BUILD_DIR/mainflux-ws &
 
+###
+# NORMALIZER 
+###
+MF_NORMALIZER_LOG_LEVEL=INFO MF_NORMALIZER_PORT=8184 MF_NATS_URL=localhost:4222 $BUILD_DIR/mainflux-normalizer &
+
+
```
Comment (drasko) [inline scripts/run.sh]: Kill this extra whiteline, one lineskip is enough.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by anovakovic01** [comment]
@mteodor Please sign your commit.

**Review by nmarcetic — APPROVED** [review]

**Comment by drasko** [comment]
@mteodor welcome to Mainflux contributors ;)

**Pull request merged**